diff --git a/dist/mermaid-isomorphic.js b/dist/mermaid-isomorphic.js
index aa5dc09a5dfb58a98d3f12cd2fc473caddd97b0d..8c764b00f7dd70006c80cad41014c1d4e818925b 100644
--- a/dist/mermaid-isomorphic.js
+++ b/dist/mermaid-isomorphic.js
@@ -1,11 +1,12 @@
+import { resolve } from 'import-meta-resolve';
 import { chromium } from 'playwright';
-const html = import.meta.resolve('../index.html');
+const html = resolve('../index.html', import.meta.url);
 const mermaidScript = {
-    url: import.meta.resolve('mermaid/dist/mermaid.js')
+    url: resolve('mermaid/dist/mermaid.js', import.meta.url)
 };
 const faStyle = {
     // We use url, not path. If we use path, the fonts can’t be resolved.
-    url: import.meta.resolve('@fortawesome/fontawesome-free/css/all.css')
+    url: resolve('@fortawesome/fontawesome-free/css/all.css', import.meta.url)
 };
 /* c8 ignore start */
 /**
diff --git a/src/mermaid-isomorphic.ts b/src/mermaid-isomorphic.ts
index 6e5720399c7e994855ca58adb29d73639e241172..025adefe72e03698bdfaa400505027ffa4f56e75 100644
--- a/src/mermaid-isomorphic.ts
+++ b/src/mermaid-isomorphic.ts
@@ -1,15 +1,16 @@
+import { resolve } from 'import-meta-resolve'
 import { type Mermaid, type MermaidConfig } from 'mermaid'
 import { type BrowserType, chromium, type LaunchOptions, type Page } from 'playwright'
 
 declare const mermaid: Mermaid
 
-const html = import.meta.resolve('../index.html')
+const html = resolve('../index.html', import.meta.url)
 const mermaidScript = {
-  url: import.meta.resolve('mermaid/dist/mermaid.js')
+  url: resolve('mermaid/dist/mermaid.js', import.meta.url)
 }
 const faStyle = {
   // We use url, not path. If we use path, the fonts can’t be resolved.
-  url: import.meta.resolve('@fortawesome/fontawesome-free/css/all.css')
+  url: resolve('@fortawesome/fontawesome-free/css/all.css', import.meta.url)
 }
 
 export interface CreateMermaidRendererOptions {
